identity/src/main/java/com/android/identity/trustmanagement/TrustManager.kt

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2023 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android.identity.trustmanagement
+
+import org.bouncycastle.asn1.x500.X500Name
+import java.security.cert.CertificateException
+import java.security.cert.PKIXCertPathChecker
+import java.security.cert.X509Certificate
+
+/**
+ * This class is used for the verification of a certificate chain
+ */
```

Comment:
Sentences should end with a period. Also elsewhere.

We should also explain - in a separate paragraph - that the user of the class can add trust roots and right now we support X509 certificates. Also a good place to explain that we use the public key in the certificate as the primary key.

Reply:
done